boilerplate/app/models/Episode.ts

```diff
@@ -18,7 +18,7 @@ export const EpisodeModel = types
   .props({
     guid: types.identifier,
     title: "",
-    pubDate: "",
+    pubDate: "", // Ex: 2022-08-12 21:05:36
```

Comment:
Eventually, it would be a good idea to handle dates how we would on a client project.

Reply:
Actually, this probably works fine, given the datePublished() view.

Reply:
Yeah, this was just a comment to remind the dev what format the date comes in from the server